cmd/lvmd/app/root.go

```diff
@@ -108,6 +109,10 @@ func subMain(ctx context.Context) error {
 	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
 	defer stop()
 
+	if config.ProfilingPort > 0 {
+		go profiling.StartProfiling(config.ProfilingPort)
```

Comment:
This goroutine leaks and is not controlled by anything. Its preferrable to add it via controller-runtimes manager.Add()

Reply:
Agreed.

Reply:
I'll get these non-controller-runtime implementations moved over to something that is able to be cancelled properly via a context/termination signal.. :+1: